src/SDKs/WebSites/WebSites.Tests/Properties/AssemblyInfo.cs

```diff
@@ -18,5 +18,5 @@
 
 [assembly: Guid("4c6e3586-2ebd-4b86-b46e-6850c1f5226e")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.0.0")]
+[assembly: AssemblyVersion("2.0.0.1")]
```

Comment:
Currently, the released package version is `1.7.1-preview` and the assembly version here is `2.0.0.0` please update the `csproj` with the new version number `2.0.0-preview` and undo these changes in `AssemblyInfo.cs`.

Reply:
Made changes.